README.md

```diff
@@ -2,7 +2,7 @@
 
 This is a handbook of coding standards and practices for WordPress that we use at Infinum. WordPress has evolved a lot since its beginnings - from a simple blog CMS to a fully fledge CMS used for a wide range of projects: simple web sites, on line shops, complex multi site projects communicating with a wide range of APIs.
```

Comment:
This is a handbook of coding standards and practices for WordPress that we use at Infinum. WordPress has evolved a lot since its beginnings—from a simple blog CMS to a full-fledged CMS used for a wide range of projects: simple websites, online shops, complex multi-site projects communicating with a wide range of APIs.